README.md

```diff
@@ -1,9 +1,19 @@
 <img src="https://www.kubeflow.org/images/logo.svg" width="100">
-Kubeflow is a Cloud Native platform for machine learning based on Google’s internal machine learning pipelines.
+Kubeflow the cloud-native platform for machine learning operations - pipelines, training and deployment.
 
 ---
 Please refer to the official docs at [kubeflow.org](http://kubeflow.org).
 
+## Working Groups
+The Kubeflow community is organized into working groups (WGs) with associated repositories, that focus on specific pieces of the ML platform. 
+
+* [AutoML](https://github.com/kubeflow/community/tree/master/wg-automl)
+* [Deployment](https://github.com/kubeflow/community/tree/master/wgdeployment)
+* [Manifests](https://github.com/kubeflow/community/tree/master/wg-manifests)
```

Comment:
We need to update those links to make sure they can persist for a long time without going back and forth to fix it.

Reply:
That is a good idea! What would be a good way to do it?

Reply:
The current links look right. kubeflow/community/master is the long term location

Reply:
> * [Deployment] (https://github.com/kubeflow/community/tree/master/wgdeployment)

This is not correct, can we make it with a dash? `wg-deployment`